features/provider/admin/account/invitations.feature

```diff
@@ -84,6 +84,7 @@ Feature: Provider Account Settings User Invitations
         | Username              | peter             |
         | Password              | 123456            |
         | Password confirmation | 123456            |
+      And they log out
```

Comment:
I am not sure why this worked before (at least sometimes). I think if the "form is submitted with" is not completed, than it works.
Otherwise, "the provider logs in" (for the already logged user) give some weird result.